src/m365/spo/commands/tenant/tenant-applicationcustomizer-get.spec.ts

```diff
@@ -0,0 +1,340 @@
+  });
+
+  beforeEach(() => {
+    log = [];
+    logger = {
+      log: (msg: string) => {
+        log.push(msg);
+      },
+      logRaw: (msg: string) => {
+        log.push(msg);
+      },
+      logToStderr: (msg: string) => {
+        log.push(msg);
+      }
+    };
+    loggerLogSpy = sinon.spy(logger, 'log');
+  });
+
+  afterEach(() => {
+    sinonUtil.restore([
+      request.get
+    ]);
+  });
+
+  after(() => {
+    sinonUtil.restore([
+      auth.restoreAuth,
+      telemetry.trackEvent,
+      pid.getProcessName
+    ]);
+    auth.service.connected = false;
+    auth.service.spoUrl = undefined;
+  });
+
+  it('has correct name', () => {
+    assert.strictEqual(command.name, commands.TENANT_APPLICATIONCUSTOMIZER_GET);
+  });
+
+  it('has a description', () => {
+    assert.notStrictEqual(command.description, null);
+  });
+
+  it('fails validation if the id is not a valid GUID', async () => {
+    const actual = await command.validate({ options: { id: 'abc' } }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('fails validation if the clientSideComponentId is not a valid GUID', async () => {
+    const actual = await command.validate({ options: { clientSideComponentId: 'abc' } }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('fails validation when all options are specified', async () => {
+    const actual = await command.validate({
+      options: {
+        title: title,
+        id: id,
+        clientSideComponentId: clientSideComponentId
+      }
+    }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('fails validation when no options are specified', async () => {
+    const actual = await command.validate({
+      options: {
+      }
+    }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('fails validation when title and id options are specified', async () => {
+    const actual = await command.validate({
+      options: {
+        title: title,
+        id: id
+      }
+    }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('fails validation when title and clientSideComponentId options are specified', async () => {
+    const actual = await command.validate({
+      options: {
+        title: title,
+        clientSideComponentId: clientSideComponentId
+      }
+    }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('fails validation when id and clientSideComponentId options are specified', async () => {
+    const actual = await command.validate({
+      options: {
+        id: id,
+        clientSideComponentId: clientSideComponentId
+      }
+    }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('passes validation if id is a valid GUID', async () => {
+    const actual = await command.validate({ options: { id: id } }, commandInfo);
+    assert.strictEqual(actual, true);
+  });
+
+  it('passes validation if title is valid', async () => {
```

Comment:
Let's rename this test to `passed validation when title specified` to better reflect its intent. Since there's no specific format for the title, it's a bit odd to say that we're expecting a valid title